packages/botfuel-dialog/src/index.js

```diff
@@ -22,6 +22,7 @@ const BotTextMessage = require('./messages/bot-text-message');
 const BotTableMessage = require('./messages/bot-table-message');
 const BotfuelAdapter = require('./adapters/botfuel-adapter');
 const BotfuelNlu = require('./nlus/botfuel-nlu');
+const BotfuelTrainerNlu = require('./nlus/botfuel-trainer-nlu');
```

Comment:
This is temporary right?
This should be named botfuel-nlu

Reply:
Yes this is temporally and will replace botfuel-nlu in the future. If we named in botfuel-nlu now then we must rename current (modified) botfuel-nlu to something else (like natural-nlu) and update this everywhere (default config + tests + may be sample bots that mentioned botfuel-nlu + sample-module)